pxteditor/experiments.ts

```diff
@@ -171,6 +171,11 @@ export function all(): Experiment[] {
             name: lf("Time Machine"),
             description: lf("Save and restore past versions of a project")
         },
+        {
+            id: "forceEnableAiErrorHelp",
+            name: lf("AI Error Explainer"),
+            description: lf("Get AI's help explaining errors in your code"),
```

Comment:
Can you create an issue for the feedback and add the following:        

    feedbackUrl: "https://github.com/microsoft/pxt/issues/xxxx",

Reply:
Sure. Created https://github.com/microsoft/pxt/issues/10694 and will add it in a separate PR, since this one is already merged.